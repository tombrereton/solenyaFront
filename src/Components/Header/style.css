
/*below here goes default css for main webpage*/
header {
  width:100%;
  min-height: 50px;
  height: 50px;
  position: fixed;
  top: 0;
  background-color: rgb(45, 45, 45);
  margin: 0px auto;
  z-index: 1;
  display: flex;
}

.headerHolder{
  height: 50px;
}

.closedMenu{
  display: none;
}

.headerHolder > .logo > .logoLink > img {
  height: 35px;
  width: 35px;
  margin: 22% 0% 0% 25%;
  display: inline-block;
}

.headerHolder > .headerTeamName > a{
  display: inline-block;
  font-family: "futura-pt";
  font-size: 25px;
  text-decoration: none;
  letter-spacing: 0.9px;
  text-align: left;
  color: #ffffff;
  position: absolute;
  top: 20%;
  left: 60px;

}

  .menuLinks{
    display: inline-block;
    position: absolute;
    top: 25%;
    right: 0;
    font-family: "futura-pt";
    font-size: 20px;
    font-weight: 500;
    letter-spacing: 0.5px;
    text-align: left;
    color: #ffffff;
  }

  .menuLinks > .webHome > a{
    display: inline-block;
    position: absolute;
    right: 90px;
    font-family: "futura-pt";
    font-size: 20px;
    font-weight: 500;
    letter-spacing: 0.5px;
    text-align: left;
    color: #ffffff;
    text-decoration: none;

  }

  .menuLinks > .webBag{
    display: inline-block;
    position: absolute;
    right: 30px;
  }

/*below is the mobile css*/
@media screen and (max-width: 520px) {
  header .burgerMenu {
    cursor: pointer;
    position: absolute;
    top: 25%;
    right: 1em;
  }

  .headerHolder > .logo > .logoLink > img  {
  height: 30px;
  width: 30px;
  margin: 9px;
}


  .bar1,
  .bar2,
  .bar3 {
    width: 25px;
    height: 2px;
    background-color: #ffffff;
    margin: 21% 15% 2% 0%;
    transition: 0.4s;
  }

  .change .bar1 {
    -webkit-transform: rotate(-45deg) translate(-9px, 4px);
    transform: rotate(-45deg) translate(-5px, 6px);
  }

  .change .bar2 {
    opacity: 0;
  }

  .change .bar3 {
    -webkit-transform: rotate(45deg) translate(-5px, -4px);
    transform: rotate(45deg) translate(-5px, -5px);
  }

  .openMenu {
    position: relative;
    height: 35px;
    width: 100%;
    position: fixed;
    z-index: 3;
    top: 46px;
    left: 0;
    background-color: #2d2d2d;
    overflow: hidden;
    transition: 0;
  }

  .closedMenu {
    width: 0;
  }

  .closedMenu a {
    display: none;
  }

  .openMenu a {
    padding-top: 0;
    text-decoration: none;
    font-family: "futura-pt";
    font-size: 18px;
    font-weight: 400;
    color: #ffffff;
    transition: 0.3s;

  }

  .homeLink {
    width:49%;
    text-align: center;
    border-right-style: solid;
    border-color: #ffffff;
    border-width: thin;
  }

  .bagLink {
    width:49%;
    text-align: center;
  }

  .logo{
    margin: 2% 0% 0% 2%;
  }

  .headerHolder > .headerTeamName{
    display: none;
  }
  
  .headerHolder > .menuLinks{
    display: none;
  }

}