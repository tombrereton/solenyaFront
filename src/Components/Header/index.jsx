import React from "react";
import {BrowserRouter as Router, Link} from "react-router-dom";
import Plp from "../Plp";
import "./style.css";

class Header extends React.Component  {
    constructor(props){      
        super(props)
        this.state = {
            toggled: false
        }
    }

    onButtonClicked(){
        this.setState({
            toggled: !this.state.toggled
        })
        console.log("click happened");
    }

    onLinkClicked(){
        this.setState({
            toggled: !this.state.toggled
        })
        newrelic.addPageAction("clickToHome")
    }

    render (){ 
        const toggled = this.state.toggled;
        const toggledClass = "container" + (toggled ? " change" : "");
        const menuClass = (toggled ? "openMenu" : "closedMenu");

        return(
            <header>     
                    <div className={toggledClass}  onClick={() => this.onButtonClicked()}>
                        <div className="bar1"></div>
                        <div className="bar2"></div>
                        <div className="bar3"></div>          
                    </div> 
                    <div className={menuClass}>                        
                        <div>
                                    <Link  className = "home-link" to="/" onClick={() => this.onLinkClicked()}>
                                        Home
                                    </Link> 
                        </div>
                    </div>  

            </header > 
            );
            
        }

    

}


    export default Header;

