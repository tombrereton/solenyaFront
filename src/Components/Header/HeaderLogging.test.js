import React from "react";

import Header from "../Header/index.jsx";

import Enzyme, { shallow } from "enzyme";

import Adapter from "enzyme-adapter-react-15";

Enzyme.configure({ adapter: new Adapter() });

console.error = message => {
  throw new Error(message);
};

describe("Link Logging", () => {
  it("It should match snapshot", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });

  it("Should have a link tag", () => {
    const wrapper = shallow(<Header />);
    const linkTag = wrapper.find("Link");
    expect(linkTag.exists()).toBe(true);
  });
});
