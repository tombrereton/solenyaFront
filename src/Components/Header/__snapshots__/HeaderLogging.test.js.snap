// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link Logging It should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="headerHolder"
>
          <div
                    className="logo"
          >
                    <Link
                              className="logoLink"
                              onClick={[Function]}
                              replace={false}
                              to="/"
                    >
                              <img
                                        alt="logo"
                                        src="https://i.imgur.com/zkkUfhc.png"
                              />
                    </Link>
          </div>
          <div
                    className="headerTeamName"
          >
                    <Link
                              className="nameLink"
                              onClick={[Function]}
                              replace={false}
                              to="/"
                    >
                              SOLENYA
                    </Link>
          </div>
          <div
                    className="menuLinks"
          >
                    <div
                              className="webHome"
                    >
                              <Link
                                        className="webHomeLink"
                                        onClick={[Function]}
                                        replace={false}
                                        to="/"
                              >
                                        PRODUCTS
                              </Link>
                    </div>
                    <div
                              className="webBag"
                    >
                              <Link
                                        className="bagLink"
                                        replace={false}
                                        to="#"
                              >
                                        BAG
                              </Link>
                    </div>
          </div>
          <div
                    className="burgerMenu"
          >
                    <div
                              className="burgerContainer"
                              onClick={[Function]}
                    >
                              <div
                                        className="bar1"
                              />
                              <div
                                        className="bar2"
                              />
                              <div
                                        className="bar3"
                              />
                    </div>
          </div>
</div>,
        <div
          className="closedMenu"
>
          <div
                    className="homeLink"
          >
                    <Link
                              className="home-link"
                              onClick={[Function]}
                              replace={false}
                              to="/"
                    >
                              Products
                    </Link>
          </div>
          <div
                    className="bagLink"
          >
                    <Link
                              className="bag-link"
                              replace={false}
                              to="#"
                    >
                              Bag
                    </Link>
          </div>
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="logo"
>
              <Link
                            className="logoLink"
                            onClick={[Function]}
                            replace={false}
                            to="/"
              >
                            <img
                                          alt="logo"
                                          src="https://i.imgur.com/zkkUfhc.png"
                            />
              </Link>
</div>,
            <div
              className="headerTeamName"
>
              <Link
                            className="nameLink"
                            onClick={[Function]}
                            replace={false}
                            to="/"
              >
                            SOLENYA
              </Link>
</div>,
            <div
              className="menuLinks"
>
              <div
                            className="webHome"
              >
                            <Link
                                          className="webHomeLink"
                                          onClick={[Function]}
                                          replace={false}
                                          to="/"
                            >
                                          PRODUCTS
                            </Link>
              </div>
              <div
                            className="webBag"
              >
                            <Link
                                          className="bagLink"
                                          replace={false}
                                          to="#"
                            >
                                          BAG
                            </Link>
              </div>
</div>,
            <div
              className="burgerMenu"
>
              <div
                            className="burgerContainer"
                            onClick={[Function]}
              >
                            <div
                                          className="bar1"
                            />
                            <div
                                          className="bar2"
                            />
                            <div
                                          className="bar3"
                            />
              </div>
</div>,
          ],
          "className": "headerHolder",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="logoLink"
                onClick={[Function]}
                replace={false}
                to="/"
>
                <img
                                alt="logo"
                                src="https://i.imgur.com/zkkUfhc.png"
                />
</Link>,
              "className": "logo",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <img
                  alt="logo"
                  src="https://i.imgur.com/zkkUfhc.png"
/>,
                "className": "logoLink",
                "onClick": [Function],
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "logo",
                  "src": "https://i.imgur.com/zkkUfhc.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="nameLink"
                onClick={[Function]}
                replace={false}
                to="/"
>
                SOLENYA
</Link>,
              "className": "headerTeamName",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "SOLENYA",
                "className": "nameLink",
                "onClick": [Function],
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": "SOLENYA",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="webHome"
>
                  <Link
                                    className="webHomeLink"
                                    onClick={[Function]}
                                    replace={false}
                                    to="/"
                  >
                                    PRODUCTS
                  </Link>
</div>,
                <div
                  className="webBag"
>
                  <Link
                                    className="bagLink"
                                    replace={false}
                                    to="#"
                  >
                                    BAG
                  </Link>
</div>,
              ],
              "className": "menuLinks",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    className="webHomeLink"
                    onClick={[Function]}
                    replace={false}
                    to="/"
>
                    PRODUCTS
</Link>,
                  "className": "webHome",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "PRODUCTS",
                    "className": "webHomeLink",
                    "onClick": [Function],
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": "PRODUCTS",
                  "type": [Function],
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    className="bagLink"
                    replace={false}
                    to="#"
>
                    BAG
</Link>,
                  "className": "webBag",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "BAG",
                    "className": "bagLink",
                    "replace": false,
                    "to": "#",
                  },
                  "ref": null,
                  "rendered": "BAG",
                  "type": [Function],
                },
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="burgerContainer"
                onClick={[Function]}
>
                <div
                                className="bar1"
                />
                <div
                                className="bar2"
                />
                <div
                                className="bar3"
                />
</div>,
              "className": "burgerMenu",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="bar1"
/>,
                  <div
                    className="bar2"
/>,
                  <div
                    className="bar3"
/>,
                ],
                "className": "burgerContainer",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "bar1",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "bar2",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "bar3",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="homeLink"
>
              <Link
                            className="home-link"
                            onClick={[Function]}
                            replace={false}
                            to="/"
              >
                            Products
              </Link>
</div>,
            <div
              className="bagLink"
>
              <Link
                            className="bag-link"
                            replace={false}
                            to="#"
              >
                            Bag
              </Link>
</div>,
          ],
          "className": "closedMenu",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="home-link"
                onClick={[Function]}
                replace={false}
                to="/"
>
                Products
</Link>,
              "className": "homeLink",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Products",
                "className": "home-link",
                "onClick": [Function],
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": "Products",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="bag-link"
                replace={false}
                to="#"
>
                Bag
</Link>,
              "className": "bagLink",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Bag",
                "className": "bag-link",
                "replace": false,
                "to": "#",
              },
              "ref": null,
              "rendered": "Bag",
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="headerHolder"
>
            <div
                        className="logo"
            >
                        <Link
                                    className="logoLink"
                                    onClick={[Function]}
                                    replace={false}
                                    to="/"
                        >
                                    <img
                                                alt="logo"
                                                src="https://i.imgur.com/zkkUfhc.png"
                                    />
                        </Link>
            </div>
            <div
                        className="headerTeamName"
            >
                        <Link
                                    className="nameLink"
                                    onClick={[Function]}
                                    replace={false}
                                    to="/"
                        >
                                    SOLENYA
                        </Link>
            </div>
            <div
                        className="menuLinks"
            >
                        <div
                                    className="webHome"
                        >
                                    <Link
                                                className="webHomeLink"
                                                onClick={[Function]}
                                                replace={false}
                                                to="/"
                                    >
                                                PRODUCTS
                                    </Link>
                        </div>
                        <div
                                    className="webBag"
                        >
                                    <Link
                                                className="bagLink"
                                                replace={false}
                                                to="#"
                                    >
                                                BAG
                                    </Link>
                        </div>
            </div>
            <div
                        className="burgerMenu"
            >
                        <div
                                    className="burgerContainer"
                                    onClick={[Function]}
                        >
                                    <div
                                                className="bar1"
                                    />
                                    <div
                                                className="bar2"
                                    />
                                    <div
                                                className="bar3"
                                    />
                        </div>
            </div>
</div>,
          <div
            className="closedMenu"
>
            <div
                        className="homeLink"
            >
                        <Link
                                    className="home-link"
                                    onClick={[Function]}
                                    replace={false}
                                    to="/"
                        >
                                    Products
                        </Link>
            </div>
            <div
                        className="bagLink"
            >
                        <Link
                                    className="bag-link"
                                    replace={false}
                                    to="#"
                        >
                                    Bag
                        </Link>
            </div>
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="logo"
>
                <Link
                                className="logoLink"
                                onClick={[Function]}
                                replace={false}
                                to="/"
                >
                                <img
                                                alt="logo"
                                                src="https://i.imgur.com/zkkUfhc.png"
                                />
                </Link>
</div>,
              <div
                className="headerTeamName"
>
                <Link
                                className="nameLink"
                                onClick={[Function]}
                                replace={false}
                                to="/"
                >
                                SOLENYA
                </Link>
</div>,
              <div
                className="menuLinks"
>
                <div
                                className="webHome"
                >
                                <Link
                                                className="webHomeLink"
                                                onClick={[Function]}
                                                replace={false}
                                                to="/"
                                >
                                                PRODUCTS
                                </Link>
                </div>
                <div
                                className="webBag"
                >
                                <Link
                                                className="bagLink"
                                                replace={false}
                                                to="#"
                                >
                                                BAG
                                </Link>
                </div>
</div>,
              <div
                className="burgerMenu"
>
                <div
                                className="burgerContainer"
                                onClick={[Function]}
                >
                                <div
                                                className="bar1"
                                />
                                <div
                                                className="bar2"
                                />
                                <div
                                                className="bar3"
                                />
                </div>
</div>,
            ],
            "className": "headerHolder",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  className="logoLink"
                  onClick={[Function]}
                  replace={false}
                  to="/"
>
                  <img
                                    alt="logo"
                                    src="https://i.imgur.com/zkkUfhc.png"
                  />
</Link>,
                "className": "logo",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <img
                    alt="logo"
                    src="https://i.imgur.com/zkkUfhc.png"
/>,
                  "className": "logoLink",
                  "onClick": [Function],
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "logo",
                    "src": "https://i.imgur.com/zkkUfhc.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  className="nameLink"
                  onClick={[Function]}
                  replace={false}
                  to="/"
>
                  SOLENYA
</Link>,
                "className": "headerTeamName",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "SOLENYA",
                  "className": "nameLink",
                  "onClick": [Function],
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": "SOLENYA",
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="webHome"
>
                    <Link
                                        className="webHomeLink"
                                        onClick={[Function]}
                                        replace={false}
                                        to="/"
                    >
                                        PRODUCTS
                    </Link>
</div>,
                  <div
                    className="webBag"
>
                    <Link
                                        className="bagLink"
                                        replace={false}
                                        to="#"
                    >
                                        BAG
                    </Link>
</div>,
                ],
                "className": "menuLinks",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      className="webHomeLink"
                      onClick={[Function]}
                      replace={false}
                      to="/"
>
                      PRODUCTS
</Link>,
                    "className": "webHome",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "PRODUCTS",
                      "className": "webHomeLink",
                      "onClick": [Function],
                      "replace": false,
                      "to": "/",
                    },
                    "ref": null,
                    "rendered": "PRODUCTS",
                    "type": [Function],
                  },
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      className="bagLink"
                      replace={false}
                      to="#"
>
                      BAG
</Link>,
                    "className": "webBag",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "BAG",
                      "className": "bagLink",
                      "replace": false,
                      "to": "#",
                    },
                    "ref": null,
                    "rendered": "BAG",
                    "type": [Function],
                  },
                  "type": "div",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="burgerContainer"
                  onClick={[Function]}
>
                  <div
                                    className="bar1"
                  />
                  <div
                                    className="bar2"
                  />
                  <div
                                    className="bar3"
                  />
</div>,
                "className": "burgerMenu",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <div
                      className="bar1"
/>,
                    <div
                      className="bar2"
/>,
                    <div
                      className="bar3"
/>,
                  ],
                  "className": "burgerContainer",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "bar1",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "bar2",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "bar3",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "div",
                  },
                ],
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="homeLink"
>
                <Link
                                className="home-link"
                                onClick={[Function]}
                                replace={false}
                                to="/"
                >
                                Products
                </Link>
</div>,
              <div
                className="bagLink"
>
                <Link
                                className="bag-link"
                                replace={false}
                                to="#"
                >
                                Bag
                </Link>
</div>,
            ],
            "className": "closedMenu",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  className="home-link"
                  onClick={[Function]}
                  replace={false}
                  to="/"
>
                  Products
</Link>,
                "className": "homeLink",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Products",
                  "className": "home-link",
                  "onClick": [Function],
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Products",
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  className="bag-link"
                  replace={false}
                  to="#"
>
                  Bag
</Link>,
                "className": "bagLink",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Bag",
                  "className": "bag-link",
                  "replace": false,
                  "to": "#",
                },
                "ref": null,
                "rendered": "Bag",
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
